Modules/Messages/src/Messages.Domain/Messages.Domain.csproj

```diff
@@ -1,6 +1,10 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
-    <ItemGroup>
+	<PropertyGroup>
+		<Nullable>enable</Nullable>
+	</PropertyGroup>
```

Comment:
Can be removed. We set this via Directory.Build.props

Reply:
Changes made.